src/domains/transaction/pages/SendTransfer/SendTransfer.tsx

```diff
@@ -179,7 +181,7 @@ export const SendTransfer = () => {
 
 		const remaining = remainingBalance.minus(fee);
 
-		setValue("displayAmount", remaining.toHuman());
+		setValue("displayAmount", remaining.toHuman(decimals));
```

Comment:
The decimals should never be used manually by DW. If there is an issue with normalising this when calling `toHuman()` then this would indicate an issue in the SDK.